### ISSUE TITLE
This Device in .alexa.devices.txt causes issue

### ISSUE THREAD

**Issue by rcjokibbe** [thorsten-gehrig/alexa-remote-control#106]
I use the alexa-remote-control's speak command to send tts to certain Dots.   I have This Device in the .alexa.devicelist.txt file.  I don't know what it is, but possibly an Android phone.   When This Device is present in the .txt file the tts is sent to the wrong Dot.  The serial number shown in the command output is the device one above listed in the txt.  If I edit the appDeviceList value for This Device the speak commands are sent to the correct Dots.

Speak commands are sent to the wrong Dots
devices:[accountName|This Device
appDeviceList:[deviceAccountId|xxxxxxxxxxxxxxxxxxxx
deviceType|xxxxxxxxxxxxxx
serialNumber|xxxxxxxxxxxxxxxxxxxxxxxxxxxx]

Speak commands are sent to correct Dots
devices:[accountName|This Device
appDeviceList:[]

**Comment by adn77** [comment]
@rcjokibbe seems like you are using the plain version of the script.

It's really hard to parse the JSON device list. My logic is to assume that we always have
 accountName|dev1
 deviceType|dev1
 serialNumber|dev1
 accountName|dev2
 deviceType|dev2
 serialNumber|dev2
 etc.

Hence, by determining the index of a device with "accountName=dev2" (index 2) would give me serialNumber and deviceType.
I noticed some App-devices having multiple serialNumber entries in the list... :(

If you're not changing your devices you could probably work with a hand curated devicelist.txt.

I am open for suggestions!
Alex

**Comment by adn77** [comment]
The non-JQ version is now deprecated as JQ should be available on most platforms.
